packages/firestore/rollup.config.exp.js

```diff
@@ -36,7 +37,7 @@ const nodePlugins = [
         target: 'es2017'
       }
     },
-    clean: true,
+    cacheDir: tmp.dirSync(),
```

Comment:
Does it work? seems it creates a new dir every time, so the cache is not shared across builds.

Reply:
I believe we already didn't take advantage of cached build because we had to use "clean". IIRC, the build cache didn't work for us since we do not always use the same transformers, and the cache was not aware of what transformers were used to build it. If I leave "clean: true" but use the same cache dir, the builds fail as the clobber one directory.